pkg/build/assemble.go

```diff
@@ -172,13 +174,31 @@ func imagePlatform(img ggcrv1.Image) (*ggcrv1.Platform, error) {
 	}, nil
 }
 
+// retryGet will keep trying to get a digest for 5 seconds until it succeeds. This is specifically used for digests
+// we just created. For example, in ECR this call will sometimes return 404 while I assume S3 is becoming eventually
+// consistent. It is possible that you do GET and find the response and then do GET and get a 404. So just
+// keep trying until we get it.
+func retryGetImage(d name.Digest, opts []remote.Option) (result ggcrv1.Image, err error) {
+	for i := 0; i < 5; i++ {
```

Comment:
Are we sure 5 seconds is long enough? Maybe we could up it to more than we need just in case any network issues occur.

Reply:
Oop, this merged while I was reviewing. Not a huge thing regardless.